lib/private/message_settings_update.php

```diff
@@ -70,9 +70,29 @@ function generateQueryStringText( $CurrentValues, &$BindValues, $ValueName, $New
     
         return '';
     }
-    
+
     // -----------------------------------------------------------------------------
-    
+
+    function getRequestValue( $aRequest, $sParameterName, $sValueType = "IntValue" )
+    {
+        if (is_array($aRequest) && array_key_exists($sParameterName, $aRequest))
+        {
+            return $sParameterName;
```

Comment:
...and while we're at it - if there already is a function in place checking the request intval and strval should be involved here. Another good point for having two functions btw.